checkmarx/tablefmt.py

```diff
@@ -0,0 +1,181 @@
+import tabulate
+import textwrap
+import typing
+
+import checkmarx.client
+import checkmarx.model as model
+
+
+def get_scan_info_table(
+    scan_results: typing.Iterable[model.ScanResult],
+    tablefmt: str = 'simple',
+):
+    scan_info_header = ('ScanId', 'ArtifactName', 'ScanState', 'Start', 'End')
```

Comment:
Camel-case in table-headers? Not sure about that.
Consider using spaces.
```suggestion
    scan_info_header = ('Scan ID', 'Artifact Name', 'Scan State', 'Start', 'End')
```